BraintreeCore/src/main/java/com/braintreepayments/api/IntegrationType.kt

```diff
@@ -0,0 +1,14 @@
+package com.braintreepayments.api
+
+import androidx.annotation.StringDef
+import java.lang.annotation.Retention
+import java.lang.annotation.RetentionPolicy
+
+internal object IntegrationType {
```

Comment:
This needs to be public with `@RestrictTo` as well to be accessible from DropIn.